system-int-test/test/cmr/system_int_test/ingest/translation_test.clj

```diff
@@ -47,19 +47,28 @@
   <Visible>true</Visible>
   </Collection>")
 
+(defn- convert-to-sets
+  "Convert lists in the umm record to sets so order doesn't matter during comparison"
+  [record]
+  (-> record
```

Comment:
Fair. I had wondered if that was the case, but the name and docstring led me to believe that it wasn't the case. Forget I said anything 😄